pyvista/themes.py

```diff
@@ -150,12 +150,11 @@ def set_plot_theme(theme):
         except KeyError:
             raise ValueError(f"Theme {theme} not found in PyVista's native themes.")
         pyvista.global_theme.load_theme(new_theme_type())
-    elif isinstance(theme, DefaultTheme):
+    elif isinstance(theme, Theme):
         pyvista.global_theme.load_theme(theme)
     else:
         raise TypeError(
-            f'Expected a ``pyvista.themes.DefaultTheme`` or ``str``, not '
-            f'a {type(theme).__name__}'
+            f'Expected a ``pyvista.themes.Theme`` or ``str``, not ' f'a {type(theme).__name__}'
```

Comment:
```suggestion
            f'Expected a ``pyvista.themes.Theme`` or ``str``, not {type(theme).__name__}'
```
(Removing the article to make it insensitive to the first letter of the type name.)